src/views/Error.vue

```diff
@@ -1,10 +1,13 @@
 <template>
   <div class="container">
+    <p style="font-size:5em">&#128546;</p>
+
     <!-- 404 error starts -->
     <div v-if="isPageNotFound">
-      <h1>404</h1>
-      <p>You have visited a wrong link</p>
-      <p>आप गलत लिंक पर आ गए हैं</p>
+      <h2>आपसे गलत लिंक कॉपी हो गया है | आप वापस जाकर सही लिंक यहाँ कॉपी करके पेस्ट करिए | लिंक कुछ ऐसा दिखेगा:</h2>
+      <h2 style="color:blue">https://player.plio.in/#/play/adndmidoe</h2>
+      <h2>You have copied a wrong link. Please go back and copy the link again. The link would look something like this:</h2>
+      <h2 style="color:blue">https://player.plio.in/#/play/adndmidoe</h2>
```

Comment:
Especailly since both the `h2`s share a style

Reply:
haan makes sense. Karta!